fuel-block-importer/src/service.rs

```diff
@@ -0,0 +1,48 @@
+use crate::Config;
+use fuel_core_interfaces::block_importer::{ImportBlockBroadcast, ImportBlockMpsc};
+use parking_lot::Mutex;
+use tokio::{
+    sync::{broadcast, mpsc},
+    task::JoinHandle,
+};
+
+pub struct Service {
+    join: Mutex<Option<JoinHandle<()>>>,
+    sender: mpsc::Sender<ImportBlockMpsc>,
+    broadcast: broadcast::Sender<ImportBlockBroadcast>,
+}
+
+impl Service {
+    pub async fn new(_config: &Config, _db: ()) -> Result<Self, anyhow::Error> {
+        let (sender, _receiver) = mpsc::channel(100);
+        let (broadcast, _receiver) = broadcast::channel(100);
+        Ok(Self {
+            sender,
+            broadcast,
+            join: Mutex::new(None),
+        })
+    }
+
+    pub async fn start(&self) {
+        let mut join = self.join.lock();
+        if join.is_none() {
+            *join = Some(tokio::spawn(async {}));
+        }
+    }
+
+    pub async fn stop(&self) -> Option<JoinHandle<()>> {
+        let join = self.join.lock().take();
+        if join.is_some() {
+            let _ = self.sender.send(ImportBlockMpsc::Stop);
+        }
+        join
```

Comment:
I feel like these Stop messages are not necessary and don't add much. We could simply abort the task:

```suggestion
        let join = self.join.lock().take();
        if let Some(handle) = join {
            handle.abort();
        }
        join
```

Reply:
`Stop` allow the module to gracefully finish whatever it was working on or do cleanup if needed.